cadctl/cmd/investigate/investigate.go

```diff
@@ -138,20 +138,13 @@ func run(_ *cobra.Command, _ []string) error {
 		return ccam.Evaluate(cluster, err, ocmClient, pdClient, alertTypeString)
 	}
 
-	logging.Info("Successfully jumpRoled into the customer account. Removing existing 'Cloud Credentials Are Missing' limited support reasons.")
-	err = ccam.RemoveLimitedSupport(cluster, ocmClient, pdClient, alertTypeString)
-	if err != nil {
-		return err
-	}
-
 	investigationResources := &investigation.Resources{AlertType: alertType, Cluster: cluster, ClusterDeployment: clusterDeployment, AwsClient: customerAwsClient, OcmClient: ocmClient, PdClient: pdClient}
 
 	run := investigation.NewInvestigation()
 
 	switch alertType {
 	case investigation.ClusterHasGoneMissing:
 		run.Triggered = chgm.InvestigateTriggered
-		run.Resolved = chgm.HandleResolved
```

Comment:
when we roll out this change to servicelogs, do we need a transition to remove all the LS reasons that are currently applied to triggered chgm's?

edit: just checked https://github.com/openshift/configuration-anomaly-detection/pull/245/files#diff-f8afe34e2c404d34ac56578a6d1617797fd3029613be45ff0f688b21f315f5e3R477 
My understanding is, we will blindly remove LS from resolving chgm. This will slowly remove LS reasons from "old" chgm alerts, regardless of actual LS. Once we no longer have old chgm we can remove the LS removal logic as well. 

Might be worth to keep the handleResolved here (should not interfere with the rest of the change?), switch to servicelogs and remove this line further down the line, to catch as many of the old alerts as possible?
However, removing it would work as well with the caveat that we increase manual LS removal load for MCS.

Reply:
> However, removing it would work as well with the caveat that we increase manual LS removal load for MCS.
Given that most CHGM alerts are very likely clusters that have already been wiped and/or internal e2e clusters, I think this variant is totally fine.